lib/matplotlib/cbook/deprecation.py

```diff
@@ -100,6 +100,7 @@ def warn_deprecated(
     """
     message = _generate_deprecation_message(
         since, message, name, alternative, pending, obj_type, removal=removal)
+    message = '\n' + message
```

Comment:
Minor style thing: I would not do this explicitly on a separate line, but rather:
`warnings.warn('\n' + message, ...)`

Reply:
Why, out of interest?

Reply:
Because you are making assumptions about the width of the window of the output (most of the time a normal linewrap will just work as well); moreover people can for example capture warnings into the logging system (logging.captureWarnings) and do all kinds of custom formatting of the message there as well.
Not going to block the PR over that, just giving my opinion though.

Reply:
I think of it as making assumptions about the maximum number of characters that should reasonably fit on one line, with the linebreak making the message as a whole more readable. I'd be inclined to say that if advanced users want to do custom formatting then it's much easier to remove line breaks than try and work out where to put them in.